src/components/views/rooms/RoomTile2.tsx

```diff
@@ -68,6 +80,10 @@ interface IState {
     generalMenuPosition: PartialDOMRect;
 }
 
+type Volume = ALL_MESSAGES_LOUD | ALL_MESSAGES | MENTIONS_ONLY | MUTE;
```

Comment:
This should be defined elsewhere, not here.

Reply:
There is currently no sane place to put it, RoomNotifs is JS

Reply:
we can do something similar to the early dispatcher work: `RoomNotifTypes.ts` with a comment saying `// TODO: The rest`.

Reply:
RoomNotif is probably dying as part of FTUE Notifications though :((

Reply:
`Volume.ts` then? It's just too wrong to leave it here (it's bad enough there's membership handling utils in the room list store, we shouldn't really be pushing bad patterns through)

Reply:
Volume as a concept is being nuked then but sure will place it somewhere.